docs: fix link to plugins page source

ayastreb opened the pull request:
The link to plugins source page is broken in the docs and leads to 404 error page.
I've changed it from relative URL to absolute URL.

parkr commented:
@ayastreb When you run `bundle exec rake site:generate` locally, this is clobbered because this file comes from `.github/CONTRIBUTING.md`. We should change it there instead!

ayastreb commented:
@parkr I reverted `site/_docs/contributing.md` and moved this fix to `.github/CONTRIBUTING.md`, now `bundle exec rake site:generate` runs ok, but it changes the `site/_docs/contributing.md` localy 😄 
Should I change it in both files and commit?

parkr commented:
> Should I change it in both files and commit? 

Yes please! And squash into one commit please :smile:

ayastreb commented:
Done!

parkr commented:
LGTM. 😄 

@jekyllbot: merge +site

ayastreb commented:
@parkr, hm... the link is still broken here - https://jekyllrb.com/docs/contributing/
do you need to deploy docs fixes manually or will it get there automatically somehow? 😄

parkr commented:
@ayastreb It's manual - haven't done it yet!
